models/bow/__main__.py

```diff
@@ -13,46 +13,54 @@
     args = get_args() # extracts all relevant variables for BoW training
 
     # Setup config
-    config = deepcopy(args)
+    config = deepcopy(args) # Copy the configuration to avoid changing the original 'args'
```

Comment:
Kind of picky on my part, but try to keep comments above the code instead of on the same line to maximize readibility. Unless its a super short two word comment

Reply:
would look over and do this for all the comments